src/frontend/src/header.tsx

```diff
@@ -219,6 +212,14 @@ const UserSection = ({ user }: { user: UserType }) => {
                             >
                                 <Journal /> JOURNAL
                             </a>
+                            <a
```

Comment:
Did you check this on mobile? I'm afraid this would be too many tiles for mobiles screen now. There was a reason, why the name was above 😅 Also because it is one of the most often clicked links for users checking their rewards (or was in the past).